### PR TITLE
Add Telemetry to PBA

### PR DESCRIPTION
**Pull request by htdvisser**
#### Summary

This PR adds Prometheus telemetry to the Packet Broker Agent.

#### Testing

⚠️ I haven't tested this. Was hoping that @johanstokking could do that.

##### Regressions

None expected. Worst case it just doesn't work.

#### Notes for Reviewers

⚠️ @johanstokking please test this

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [ ] Documentation: Relevant documentation is added or updated.
- [x] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Review by adriansmares — APPROVED** [review]
This looks good to me (did not test).

I suggest moving observations to `observe*` functions, in case we think about adding events somewhere in the future.

File: pkg/packetbrokeragent/agent.go [hunk 1]
```diff
@@ -763,6 +775,16 @@ func (a *Agent) handleUplinkMessage(ctx context.Context, up *packetbroker.Routed
 			return err
 		}
 	}
+
+	var forwarderNetID types.NetID
+	forwarderNetID.UnmarshalNumber(up.ForwarderNetId)
+	var homeNetworkNetID types.NetID
+	homeNetworkNetID.UnmarshalNumber(up.ForwarderNetId)
```
Comment (johanstokking) [inline pkg/packetbrokeragent/agent.go]:
```suggestion
	homeNetworkNetID.UnmarshalNumber(up.HomeNetworkNetId)
```

**Review by johanstokking — CHANGES_REQUESTED** [review]
Will test, needs small fix though

File: pkg/packetbrokeragent/agent.go (hunk 1 above)
Comment (johanstokking) [reply pkg/packetbrokeragent/agent.go]: It's Friday, it's ok

**Comment by htdvisser** [comment]
> I suggest moving observations to `observe*` functions, in case we think about adding events somewhere in the future.

I prefer to keep things simple until we actually decide to add those events.

**Review by johanstokking — APPROVED** [review]
Tested and works

**Comment by johanstokking** [comment]
```
# HELP ttn_lw_pba_downlink_forwarded_total Total number of downlinks forwarded to Packet Broker
# TYPE ttn_lw_pba_downlink_forwarded_total counter
ttn_lw_pba_downlink_forwarded_total{forwarder_cluster_id="",forwarder_net_id="000000",forwarder_tenant_id="johan",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 1
# HELP ttn_lw_pba_downlink_received_total Total number of downlinks received from Packet Broker
# TYPE ttn_lw_pba_downlink_received_total counter
ttn_lw_pba_downlink_received_total{forwarder_cluster_id="",forwarder_net_id="000000",forwarder_tenant_id="johan",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 1
# HELP ttn_lw_pba_uplink_forwarded_total Total number of uplinks forwarded to Packet Broker
# TYPE ttn_lw_pba_uplink_forwarded_total counter
ttn_lw_pba_uplink_forwarded_total{forwarder_cluster_id="",forwarder_net_id="000000",forwarder_tenant_id="johan"} 1
# HELP ttn_lw_pba_uplink_received_total Total number of uplinks received from Packet Broker
# TYPE ttn_lw_pba_uplink_received_total counter
ttn_lw_pba_uplink_received_total{forwarder_cluster_id="",forwarder_net_id="000000",forwarder_tenant_id="johan",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 1
ttn_lw_pba_uplink_received_total{forwarder_cluster_id="ttn-eu1",forwarder_net_id="000013",forwarder_tenant_id="ttn",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 21
ttn_lw_pba_uplink_received_total{forwarder_cluster_id="ttn-v2-au",forwarder_net_id="000013",forwarder_tenant_id="ttn",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 87
ttn_lw_pba_uplink_received_total{forwarder_cluster_id="ttn-v2-brazil",forwarder_net_id="000013",forwarder_tenant_id="ttn",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 6
ttn_lw_pba_uplink_received_total{forwarder_cluster_id="ttn-v2-eu-1",forwarder_net_id="000013",forwarder_tenant_id="ttn",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 79
ttn_lw_pba_uplink_received_total{forwarder_cluster_id="ttn-v2-eu-2",forwarder_net_id="000013",forwarder_tenant_id="ttn",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 34
ttn_lw_pba_uplink_received_total{forwarder_cluster_id="ttn-v2-eu-3",forwarder_net_id="000013",forwarder_tenant_id="ttn",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 238
ttn_lw_pba_uplink_received_total{forwarder_cluster_id="ttn-v2-eu-4",forwarder_net_id="000013",forwarder_tenant_id="ttn",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 385
ttn_lw_pba_uplink_received_total{forwarder_cluster_id="ttn-v2-switch",forwarder_net_id="000013",forwarder_tenant_id="ttn",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 13
ttn_lw_pba_uplink_received_total{forwarder_cluster_id="ttn-v2-us-west",forwarder_net_id="000013",forwarder_tenant_id="ttn",home_network_cluster_id="johan-dev",home_network_net_id="000000",home_network_tenant_id="johan"} 92
```

**Pull request merged**